ext/Logger.php

```diff
@@ -52,7 +54,7 @@ class Logger extends Extension
     /**
      * @var \Monolog\Logger
      */
-    protected $logger;
+    protected static $logger;
```

Comment:
Sure, I don't believe this to be a BC break as it was a protected method with no getter, so it was only ever an internal property. I've changed it to static so that you can actually use the logger in your tests, and not just codeception using it internally. It's also done statically so it's a singleton and doesn't need to create a new instance every time.